components/match2/wikis/omegastrikers/match_group_input_custom.lua

```diff
@@ -25,6 +26,7 @@ local STATUS_TO_WALKOVER = {FF = 'ff', DQ = 'dq', L = 'l'}
 local NOT_PLAYED = {'skip', 'np'}
 local MAX_NUM_OPPONENTS = 2
 local MAX_NUM_PLAYERS = 10
+local NOW = os.time(os.date('!*t') --[[@as osdateparam]])
```

Comment:
should we potentionally add a `DateExt.now` (and `DateExt.today`)?
could use it in several modules across the board

Reply:
Yeah leaving OOS